frontend/packages/operator-lifecycle-manager/src/components/operand.tsx

```diff
@@ -386,7 +386,10 @@ export const OperandDetails = connectToModel((props: OperandDetailsProps) => {
     ownedDefinitions.concat(reqDefinitions),
     (def) => def.name.split('.')[0] === props.kindObj.plural,
   );
-  const statusDescriptors = _.get<Descriptor[]>(thisDefinition, 'statusDescriptors', []);
+  const statusDescriptors = _.get<Descriptor[]>(thisDefinition, 'statusDescriptors', []).filter(
+    // exclude Conditions since they are included in their own section
+    (descriptor) => descriptor.displayName !== 'Conditions',
```

Comment:
We should check `path` rather than `displayName`

```suggestion
    (descriptor) => descriptor.path !== 'conditions',
```

Reply:
We might want to skip our generic conditions handling if there's a status descriptor that's not `urn:alm:descriptor:io.kubernetes.conditions`. I wonder if we should just remove that and ask operators to specify the descriptor.

Reply:
Updated.  Thanks, @spadgett